internal/render/container.go

```diff
@@ -254,3 +264,43 @@ func (c ContainerRes) GetObjectKind() schema.ObjectKind {
 func (c ContainerRes) DeepCopyObject() runtime.Object {
 	return c
 }
+
+// Container returns the underlying container or init container
+func (c ContainerRes) Container() *v1.Container {
+	if c.IsInit {
+		if c.Index >= 0 && c.Index < len(c.pod.Spec.InitContainers) {
+			return &c.pod.Spec.InitContainers[c.Index]
+		}
+		return nil
+	}
+	if c.Index >= 0 && c.Index < len(c.pod.Spec.Containers) {
+		return &c.pod.Spec.Containers[c.Index]
+	}
+	return nil
+}
+
+// Status returns the underlying container or init container status
+func (c ContainerRes) Status() *v1.ContainerStatus {
+	if c.IsInit {
+		if c.Index >= 0 && c.Index < len(c.pod.Status.InitContainerStatuses) {
+			return &c.pod.Status.InitContainerStatuses[c.Index]
+		}
+		return nil
+	}
+	if c.Index >= 0 && c.Index < len(c.pod.Status.ContainerStatuses) {
+		return &c.pod.Status.ContainerStatuses[c.Index]
+	}
+	return nil
+}
+
+// IndexLabel returns a label for the index with a prefix for the container type.
+// Sorts so init containers are first.
+func (c ContainerRes) IndexLabel() string {
+	var containerType string
+	if c.IsInit {
+		containerType = "\u24D8" // encircled i (sorts first)
```

Comment:
Not keen on using an ascii code to distinguish and sort on init. I think `I` prefix should do.